packages/react/src/primitives/Collection/Collection.tsx

```diff
@@ -51,6 +51,7 @@ export const Collection = <Item,>({
 }: CollectionProps<Item>): JSX.Element => {
   const [searchText, setSearchText] = useState<string>();
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   const onSearch = useCallback(debounce(setSearchText, TYPEAHEAD_DELAY_MS), [
```

Comment:
@hvergara Do you have any concerns with ignoring the rules of hooks error message on this code? Judging by the code, I don't think there is a risk of stale closures here, given it only uses the `setSearchText` function, and `TYPEAHEAD_DELAY_MS` is a constant.
![Screen Shot 2021-12-07 at 2 37 31 PM](https://user-images.githubusercontent.com/6165315/145129469-372e171d-31d6-48c5-afd5-35ad8c0b49c8.png)

Reply:
I'm good